api/v1alpha1/system_types.go

```diff
@@ -208,6 +208,63 @@ var (
 			corev1.ResourceMemory: resource.MustParse("2Gi"),
 		},
 	}
+	systemDefaultSystemTektonTasks []SystemTektonTaskSpec = []SystemTektonTaskSpec{
+		{
+			Name:        pointer.String("system-backend-sync"),
+			Description: pointer.String("Runs the Backend Synchronization task"),
+			Config: &SystemTektonTaskConfig{
+				Command: []string{"container-entrypoint"},
+				Args: []string{
+					"bundle",
+					"exec",
+					"rails",
+					"backend:storage:enqueue_rewrite",
+				},
+			},
+		},
+		{
+			Name:        pointer.String("system-db-migrate"),
+			Description: pointer.String("Runs the Database Migration task"),
+			Config: &SystemTektonTaskConfig{
+				Command: []string{"container-entrypoint"},
+				Args: []string{
+					"bundle",
+					"exec",
+					"rails",
+					"backend:storage:enqueue_rewrite",
```

Comment:
I guess this is wrong, isn't it? it contains exact data as Backend Synchronization task

Reply:
Fixed in https://github.com/3scale-ops/saas-operator/pull/268/commits/272f27fadcad82a506574972fa9ba9a9fe2c4778